core/arch/aarch64/codec.txt

```diff
@@ -123,6 +124,7 @@
 ----------xxxxxxxxxxxxxxxxx-----  mem12q     # size is 16 bytes
 ----------xxxxxxxxxxxxxxxxx-----  prf12      # size is 0 bytes (prefetch variant of mem12)
 ---------x----------------------  imm12sh    # shift for ADD/SUB (immediate); '0x'
+---------x----------------------  fsz        # element width of FP vector reg (single (0x1) and double (0x3) encoding)
```

Comment:
Is the comment still correct? Could it be shorted to no more than 90 chars? (I know the 90-char rule is necessarily broken elsewhere in this file.)

Reply:
Updated, I've split it up across 2 lines, as I think it is helpful to have more details here.